code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -431,10 +429,12 @@
 	if(stat != DEAD && !(HAS_TRAIT(src, TRAIT_KNOCKEDOUT) || IsStun() || IsParalyzed() || low_power_mode)) //Not dead, not stunned.
 		if(!eye_lights)
 			eye_lights = new()
-		if(lamp_intensity > 2)
+		if(lamp_enabled)
```

Comment:
Test the borg in the darkness, on all the intensity levels.
Does the toggling always work fine? Do they look okay?
Is there no situation when the borg has light but its icon_state or overlays does not reflect it, chiefly on the values between it being off and when it just got turned on?